src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import PeopleTable from './PeopleTable';
+import ErrorMessage from './ErrorMessage';
+import NotPeople from './NotPeople';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
The initial state for 'people' is set to 'null', but it would be better to initialize it as an empty array. This would simplify the checks you have to perform later in the code (e.g., line 51) and reduce the need for optional chaining.